action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'pypi-publish'
+description: 'Upload Python distribution packages to PyPI'
+inputs:
```

Comment:
I think we need to also support a custom package index URL

Reply:
Was this supported before?

Reply:
Yes, it's natively supported by Twine using the [`TWINE_REPOSITORY_URL` env var](https://twine.readthedocs.io/en/latest/#environment-variables). I guess I just didn't explicitly put it into the README. So since the old Actions version allowed setting any env vars it was supported before.